### PR TITLE
#237 complete sa three w transfo open ended branch

### LINKED ISSUE #237: Allow contingencies on open-ended branches and three-winding transformers
Initial implementation of Security Analysis simulations (#132) did not consider contingencies on open-ended branches or three-winding transformers.

### PR DESCRIPTION
**Pull request by marcochiaramello**
Closes #237

### CODE REVIEW

File: scripts/checkContingencies.py [hunk 1]
```diff
@@ -29,25 +57,11 @@ def check_contingencies(contingencies_file, results_folder):
...
             # This is a valid contingency, check that every file exists and
             # conforms to what we expect
             for element in contingency_elements:
-                # Let's check them ahead, so that 'and' doesn't shortcut
-                c_dyd        = check_file_with(check_dyd,                     "Dyd",        dyd_file, element, contingency_id)
-                c_par        = check_file_with(check_par,                     "Par",        par_file, element, contingency_id)
-                if buildType == "Debug":
-                    c_timeline   = check_file_with(check_timeline,           "Timeline",   timeline_file, element, contingency_id)
-                c_finalState = check_file_with(check_finalState, "Final State IIDM", finalState_file, element, contingency_id)
-                print("  check dyd         : {}".format(c_dyd))
```
Comment (rosiereflo) [inline scripts/checkContingencies.py:46]: This log is lost

**Review by rosiereflo — CHANGES_REQUESTED** [review]
Unit tests?

**Comment by marcochiaramello** [comment]
> Unit tests?
Difficult to add specific unit tests for those cases. The main_sa test has been enriched to take into account those objects

File: scripts/checkContingencies.py (hunk 1 above)
Comment (marcochiaramello) [reply scripts/checkContingencies.py:46]: thx, done

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**